test/unit/dns.server.test.ts

```diff
@@ -194,5 +195,9 @@ describe('DNS server lib function test', () => {
     expect(isValueValid(DnsRecordType.AAAA, 'a:b:c:d:e:f:0:')).toBe(false);
     expect(isValueValid(DnsRecordType.AAAA, 'g:g:g:g:g:g:g:g')).toBe(false);
     expect(isValueValid(DnsRecordType.CNAME, '')).toBe(false);
+    expect(isValueValid(DnsRecordType.CNAME, '192.168.0.0')).toBe(false);
+    expect(isValueValid(DnsRecordType.CNAME, '2001:db8:3333:4444:5555:6666:7777:8888')).toBe(false);
+    expect(isValueValid(DnsRecordType.CNAME, 'improper-domain')).toBe(false);
```

Comment:
I was just wondering how the word "improper" validates this correctly?

Reply:
`CNAME` can be accepted as long as it is a proper domain format and it's not IP address. Above test fails only because it's not a domain.